appeals-service-api/src/__tests__/appeals.spec.js

```diff
@@ -346,6 +346,32 @@ describe('Appeals API', () => {
     expect(response.statusCode).toBe(400);
   });
 
+  test('PUT /api/v1/appeals/{id} - It responds with an error - the appellant owning the whole site but still told other owners should fail', async () => {
+    const appeal = await createAppeal();
+    appeal.appealSiteSection.siteOwnership.ownsWholeSite = true;
+    appeal.appealSiteSection.siteOwnership.haveOtherOwnersBeenTold = true;
+
+    const response = await request(app).put(`/api/v1/appeals/${appeal.id}`).send(appeal);
+    expect(response.body.code).toEqual(400);
+    expect(response.body.errors).toContain(
+      'If the appellant own the hole appeal site there is any other owners'
```

Comment:
"If the appellant owns the whole appeal site there can be no other owners"

Reply:
It was my french brain who decided so I'll go with your version thanks